config/toml.go

```diff
@@ -347,6 +347,11 @@ dial_timeout = "{{ .P2P.DialTimeout }}"
 #   2) "v1" - prioritized mempool.
 version = "{{ .Mempool.Version }}"
 
+# Recheck (default: true) defines whether CometBFT should recheck the
```

Comment:
Not present in `v0.34.x` branch, but no harm in adding here.

Reply:
If it is not implemented in 0.34.x maybe we can delete it from here?

Reply:
The flag exist, what is added here is a comment describing it.